schema/schema.go

```diff
@@ -26,6 +26,7 @@ const (
 	MediaTypeManifest     Validator     = v1.MediaTypeImageManifest
 	MediaTypeManifestList Validator     = v1.MediaTypeImageManifestList
 	MediaTypeImageConfig  Validator     = v1.MediaTypeImageConfig
+	TypeImageLayout       Validator     = v1.ImageLayoutFile
```

Comment:
#403 is never going to land. This effectively uses a map internally anyways. See my example.